src/rules/roads.py

```diff
@@ -28,20 +29,20 @@
 toaddress_right_calculation = Calculation('From Address Right', 'TOADDR_R', common.set_null_as_zero('TOADDR_R'))
 toaddress_right_calculation.triggers = [config.triggers.insert, config.triggers.update]
 
-name_calculation = Calculation('Name', 'NAME', 'Upper(\'NAME\')')
-alias_name_calculation = Calculation('Alias Name', 'A1_NAME', 'Upper(\'A1_AME\')')
-alias_alternate_name_calculation = Calculation('Alternate Alias Name', 'A2_NAME', 'Upper(\'A2_NAME\')')
+# name_calculation = Calculation('Name', 'NAME', 'Upper(\'NAME\')')
```

Comment:
**suggestion**: prefer removing the code rather than commenting it out

Reply:
gotcha.  will do.  just left it there as my next step.